scripts/pull-down-test-api-spec.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-TEST_API_DIR=$(realpath ./internal/api/testapi)
+TEST_API_DIR=$(realpath ./pkg/testapi)
```

Comment:
Suggestion: Since there will be more coming, how about `pkg/apiclients/testapi`?